scripts/cache-server.sh

```
@@ -0,0 +1,20 @@
+#! /usr/bin/env bash
+
+#start local cache server, as per https://github.com/NoahHsu/github-act-cache-server
+# build image and start a container(listen 8080 port) - changed to 8101 bc conflits
+echo $CACHE_SERVER_PATH
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)